tests/pytorch/test_lenet.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import numpy as np
+
+import daceml.onnx as donnx
+from daceml.pytorch import DaceModule
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from daceml.testing.utils import torch_tensors_close
+
+
+class LeNet(nn.Module):
+    def __init__(self):
+        super(LeNet, self).__init__()
+        self.conv1 = nn.Conv2d(1, 6, (3, 3))
+        self.conv2 = nn.Conv2d(6, 16, (3, 3))
+        self.fc1 = nn.Linear(16 * 6 * 6, 120)  # 6*6 from image dimension
+        self.fc2 = nn.Linear(120, 84)
+        self.fc3 = nn.Linear(84, 10)
+
+    def forward(self, x):
+        x = F.max_pool2d(F.relu(self.conv1(x)), 2)
+        x = F.max_pool2d(F.relu(self.conv2(x)), 2)
+
+        x = x.view(-1, 16 * 6 * 6)
+        x = F.relu(self.fc1(x))
+        x = F.relu(self.fc2(x))
+        x = self.fc3(x)
+        x = F.log_softmax(x, dim=1)
+        return x
+
+
+@pytest.mark.parametrize("conv_impl", ["pure", "im2col"])
```

Comment:
I have not seen the im2col implementation anywhere. Where is it defined?

Reply:
I removed it because it doesn't work #49.

Reply:
shouldn't we remove the parameterization too?

Reply:
I plan to re-add it, so I guess we can keep it until that happens